### ISSUE TITLE
TypeError: Key data must be an object for JWK import

### ISSUE THREAD

**Issue by bruscantini** [digi-trust/dt-cdn#47]
I'm seeing this error consistently on pages with HTTPS origins when running DigiTrust.getUser.
I'm on chrome Version 65.0.3325.162, in Mobile view, and loading the recommended "major release" version of the DigiTrust library:
https://cdn.digitru.st/prod/1/digitrust.min.js
This is preventing me from getting the DigiTrust ID for the user. 
Sample page: https://www.yachtingmagazine.com/yachting-cover-stories-2017

**Comment by samtingleff** [comment]
I'm unable to reproduce however we have a similar issue reported on safari. Is this still an issue, and is consistent? Do you only see this error in mobile view?

**Comment by bruscantini** [comment]
I see it on chrome, but when mobile view is mimicking iPhone7/8/9. I am able to reproduce it on that site (and others). Interestingly, I don't see the error on safari. Though, safari always returns {success: false}

**Comment by braizhas** [comment]
It's possible to reproduce this behaviour on Mac OS Safari if digitru.st is allowed to set cookies.

**Comment by ddagino** [comment]
I'm also seeing this error getting thrown from `https://cdn.digitru.st/prod/1.5.10/digitrust-server.min.js`:

```
Uncaught (in promise) TypeError: Key data must be an object for JWK import
    at Object.j.encrypt (digitrust-server.min.js:1)
    at messageHandler (dt.html:38)
```

**Comment by samtingleff** [comment]
I _think_ this should be fixed in 1.5.11
